portal/v2/src/ClusterDetail.tsx

```diff
@@ -37,15 +37,6 @@ const navStyles: Partial<INavStyles> = {
   },
 }
 
-
-// let customPanelStyle: Partial<IPanelStyles> = {
-//   root: { top: "40px", left: "225px" },
-//   content: { paddingLeft: 30, paddingRight: 5 },
-//   navigation: {
-//     justifyContent: "flex-start",
-//   },
-// }
-
```

Comment:
We shouldn't ever commit commented out code.

Reply:
It seems this was added already. It's not part of this PR. I will remove it.

Reply:
I had removed these commented lines of code already.